roles/openshift_node/library/oc_csr_approve.py

```diff
@@ -88,12 +88,13 @@ def csr_present_check(nodename, csr_dict):
 class CSRapprove(object):  # pylint: disable=useless-object-inheritance
     """Approves node CSRs"""
 
-    def __init__(self, module, oc_bin, kubeconfig, nodename):
+    def __init__(self, module, oc_bin, kubeconfig, nodename, run_attempts):  # pylint: disable=too-many-arguments
```

Comment:
Can't you use a default argument value
```suggestion
    def __init__(self, module, oc_bin, kubeconfig, nodename, run_attempts=1):  # pylint: disable=too-many-arguments
```
so that you don't have to change where the constructor is called when this argument is not passed/considered?